Add matrix clarifier to CODE_OF_CONDUCT.md (#34)
jargonLint/jargonLint: CODE_OF_CONDUCT.md

Instances of abusive, harassing, or otherwise unacceptable behavior may be
reported to the maintainers responsible for enforcement at:

> at alexvonme : matrix dot org

Author: Starting your account name with an at, is the quasi-universal way to state that your account is within a specific network such as twitter or mastodon. Everything else is redundant.

I corrected format specific to matrix.

reported to the maintainers responsible for enforcement at:

alexvonme at matrix dot org

> (This is not an email address, you may reach it after opening an account in matrix)

Reviewer: please add a link to it then.

People are lazy :grin:

Reviewer: @alexvonme what about this?

Author: I'll explain this one in the other platform we chat in.

Author: > please add a link to it then.

I don't think we need to. People link to, for example, their mastodon accounts without giving links to mastodon all the time. It's expected that people will end up in the designated network anyway when seeing a link like ``@example@mastodon.social``.

It's redundant to add links when the account starts with ``@`` since people assume the later part is a link at this point.

> (This is not an email address, you may reach it after opening an account in matrix dot org)

Reviewer: ```suggestion
(This is not an email address, you may reach it after opening an account on [matrix.org](https://matrix.org))
```
It's not an email a not soil spam, but a website. Let's people click on it.

Author: I actually commented on this, days before, then accidentally kept it at pending, see below for my latest comment.
